package.json

```
@@ -12,10 +12,11 @@
     "build:preview": "vite preview",
     "build:readme": "node tools/update-readme.js",
     "start": "vite",
+    "compile": "tsc",
```

We generally use `build` for the name of this script.
```suggestion
    "build": "tsc",
```